pkg/ephemeral-disk/ephemeral-disk_test.go

```diff
@@ -104,7 +105,7 @@ var _ = Describe("ContainerDisk", func() {
 				AppendEphemeralPVC(vmi, "fake-disk", "fake-pvc")
 
 				By("Creating VirtualMachineInstance disk image that corresponds to the VMIs PVC")
-				err := creator.CreateEphemeralImages(vmi)
+				err := creator.CreateEphemeralImages(vmi, &api.Domain{})
```

Comment:
can we get a test case similar to this one that verifies the mount directory for block devices